dirty_cat/super_vectorizer.py

```diff
@@ -427,7 +427,16 @@ def fit_transform(self, X, y=None):
         if self.verbose:
             print(f'[SuperVectorizer] Assigned transformers: {self.transformers}')
 
-        return super().fit_transform(X, y)
+        res = super().fit_transform(X, y)
+
+        # for the "remainder" columns, the ColumnTransformer transformers_ attribute
+        # contains the index instead of the column name so we convert it to the column name.
+        for i, tup in enumerate(self.transformers_):
+            name, enc, cols = tup  # Unpack
```

Comment:
Maybe let's do this only if len(cols) < 20, if you agree?

Reply:
Oh yes, agreed!